misk-crypto/src/main/kotlin/misk/crypto/CryptoModule.kt

```diff
@@ -72,16 +75,24 @@ class CryptoModule(
     }
   }
 
-  private class AeadProvider(val keyUri: String, val key: Key) : Provider<Aead> {
+  /**
+   * We do not support creating bare AEAD keys, only via envelope encryption.
```

Comment:
nit: Probably better to word it as "we only support Aead keys via envelope encryption"